.dictionary

```diff
@@ -115,6 +115,7 @@ linux
 logcat
 macOS
 manylinux
+md
```

Comment:
Weird -- I can't see where this is actually used, but probably no harm...

Reply:
In the `include` statements in file names. Looks like aspell isn't excluding those. I advised Dexter to just add it to the dictionary for now.